src/custom/components/analytics/events/transactionEvents.ts

```diff
@@ -71,6 +71,8 @@ export function orderAnalytics(action: OrderType, orderClass: OrderClass, label?
   if (action === 'Posted') {
     window.fbq?.('track', 'Lead')
     window.lintrk?.('track', { conversion_id: 10759522 })
+    window.twq?.('event', 'tw-oddz2-oddzb', {})
+    window.rdt?.('track', 'Purchase')
```

Comment:
ideally, it would be nice to abstract this `window.rdt` and `window.twq` into some util, like `import { sendTwitterEvent} from 'utils/twitter` 

It might help with readability.

Reply:
also, I would move the constants there, so we just use nice names for these weird event names

Reply:
Created a separate PR for this https://github.com/cowprotocol/cowswap/pull/1982 👍